exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/aggregate/HashAggTemplate.java

```diff
@@ -1175,6 +1175,7 @@ public AggIterOutcome outputCurrentBatch() {
       v.getValueVector().getMutator().setValueCount(numOutputRecords);
     }
 
+    outContainer.setRecordCount(numOutputRecords);
```

Comment:
Noticed the old code is using INFO debug level; shouldn't it be using debug?

Reply:
- I created an issue for centralizing setting the record counts here. We can probably just make VectorContainer.setRecordCount set the record count for value vectors as well instead of just itself. https://issues.apache.org/jira/browse/DRILL-6675
 - Discussed offline, I'll make the logging changes to HashAgg as part of https://issues.apache.org/jira/browse/DRILL-6253

Reply:
Thank you Tim for the due diligence!